doc/directives.py

```diff
@@ -87,3 +89,89 @@ def run(self):
         thumb = nodes.paragraph()
         self.state.nested_parse(thumbnail, self.content_offset, thumb)
         return [thumb]
+
+
+DEMO_GALLERY_TEMPLATE = """
+.. raw:: html
+
+   <a href={link} data-toggle="tooltip" title="{tooltip}">
+    <div class="card" style="width: 15rem; height: 19rem; margin: 10px;">
```

Comment:
If you increase the margin to 20px, it doesn't look that bad with only two cards:

![image](https://user-images.githubusercontent.com/2959003/101803895-fbeaff00-3b4b-11eb-8d4e-9f971d6cdae4.png)